### DISCUSSION TITLE
Updating packages

### DISCUSSION THREAD

**Discussion by KrunchMuffin** [scosman/CMSaasStarter#151] [Q&A]
Will I be in a world of hurt if I start updating some of the outdated packages?

**Comment by scosman** [comment]
I bump the deps on occasion and keep them up to date for `npm audit`. What are you seeing out of date?

**Reply by KrunchMuffin** [reply]
these are all out of date
    "@sveltejs/adapter-auto": "^3.2.0",
    "@sveltejs/kit": "^2.0.0",
    "@sveltejs/vite-plugin-svelte": "^3.1.1",
    "@tailwindcss/typography": "^0.5.13",
        "autoprefixer": "^10.4.15",
    "daisyui": "^4.7.3",
    "svelte-check": "^3.4.3",
    "tailwindcss": "^3.4.1",
    "typescript": "^5.0.0",
    "vitest": "^1.0.0"
     "resend": "^3.5.0",
     "stripe": "^13.3.0"

**Reply by scosman** [reply]
package.json define the minimum, not the current. Check out package-lock. SvelteKit for example is on `"version": "2.5.25"` which is almost latest

Re upgrading major versions: not sure how much pain each would be. If you find any "just work" we can take a PR to update.
